docs/faq.md

```diff
@@ -99,29 +150,34 @@ evolve without breaking existing clients: simply adding a field to a response wi
 Connect clients and servers will ignore unknown fields, provided that the
 underlying implementation allows us to do so.
 
+### How do I customize serialization errors returned by the Connect server?
+
+You can customize the error message by providing a different `Codec`. Code
+and details can't be customized.
```

Comment:
This is specific to connect-go. Can we clarify by moving this to the Go section?